dakara_server/library/management/commands/createworks.py

```diff
@@ -0,0 +1,340 @@
+                     "subtitle:{subtitle}, work_type:{work_type})".format(
+                         title=work_title,
+                         subtitle=work_subtitle,
+                         work_type=work_type_entry.query_name))
+
+        if self.debug:
+            self.debug_parser_work(dict_work)
+
+        # get or create works
+        if self.update_only:
+
+            try:
+                work_entry = Work.objects.get(
+                        title=work_title,
+                        work_type=work_type_entry,
+                        subtitle=work_subtitle)
+
+            except Work.DoesNotExist:
+                logger.debug("Work (title:{title}, "
+                             "subtitle:{subtitle}, work_type:{work_type})"
+                             " not found (update only).".format(
+                                 title=work_title,
+                                 subtitle=work_subtitle,
+                                 work_type=work_type_entry.query_name))
+
+                return
+
+        else:
+            work_entry, work_created = Work.objects.get_or_create(
+                    title=work_title,
+                    work_type=work_type_entry,
+                    subtitle=work_subtitle,
+                    defaults={
+                        'title': work_title,
+                        'work_type': work_type_entry,
+                        'subtitle': work_subtitle}
+                    )
+
+            if work_created:
+                logger.debug("Created work '{}'.".format(work_entry))
+
+        # get work alternative titles or create them
+        self.work_alt_title_creator.create_alternative_titles(
+                work_type_entry,
+                work_entry,
+                work_alternative_titles)
+
+        # save work in the database
+        work_entry.save()
+
+    def createworks(self):
+        """Create or update works provided."""
+        # parse the work file to get the data structure
+        try:
+            works = self.parser.parse_work(self.work_file)
+        except BaseException as exc:
+            raise CommandError("Error when reading the work"
+                               " file: {}".format(exc)) from exc
+
+        work_success = True
+        # get works or create it
+        for worktype_query_name, work_listing in works.items():
+
+            logger.debug("Get WorkType query name '{}'".format(
+                worktype_query_name))
+            # get work type
+            try:
+                work_type_entry = WorkType.objects.get(
+                        query_name=worktype_query_name
+                        )
+
+            except WorkType.DoesNotExist:
+                work_success = False
+                logger.error(
+                    "Unable to find work type query name '{}'. Use "
+                    "createworktypes command first to create "
+                    "work types.".format(worktype_query_name))
```

Comment:
On a second thought, I think you can simply put a `continue` here and drop the `else` block.